api/src/main/java/jakarta/data/page/CursoredPage.java

```diff
@@ -102,32 +102,6 @@
  * existing entities are modified or if an entity is re-added with different
  * sort criteria after having previously been removed.</p>
  *
```

Comment:
I am willing to lift the restriction requiring the WHERE clause to be enclosed in parenthesis if we keep the other restriction disallowing the query to end with other clauses like GROUP BY, HAVING, or ORDER BY.
Here is the old text of the section updated according to that:

```suggestion
 *
 * <h2>Cursor-based Pagination with {@code @Query}</h2>
 *
 * <p>Cursor-based pagination involves generating and appending additional
 * restrictions involving the key fields to the {@code WHERE} clause of the
 * query. For this to be possible, a user-provided JDQL or JPQL query must
 * end with a {@code WHERE} clause to which additional conditions may be
 * appended.</p>
 *
 * <p>Sorting criteria must be specified independently of the user-provided
 * query, either via the {@link OrderBy} annotation or, or by passing
 * {@link Sort} criteria within the {@linkplain PageRequest#sorts() page
 * request}. For example:</p>
 *
 * <pre>
 * {@code @Query("WHERE (ordersPlaced >= ?1 OR totalSpent >= ?2)")}
 * {@code @OrderBy("zipcode")}
 * {@code @OrderBy("birthYear")}
 * {@code @OrderBy("id")}
 * {@code CursoredPage<Customer> getTopBuyers(int minOrders, float minSpent,
 *                                     {@code PageRequest<Customer> pageRequest);}
 * </pre>
 *
```

Reply:
OK, that's fine by me.

Reply:
>    {@code @Query("WHERE (ordersPlaced >= ?1 OR totalSpent >= ?2)")}

But we can get rid of these parens, right?

Reply:
Yes, those parenthesis are optional now. I'll take them out of the example.